src/main/java/io/nats/client/JetStreamManagement.java

```diff
@@ -74,6 +74,17 @@
      */
     StreamInfo getStreamInfo(String streamName) throws IOException, JetStreamApiException;
 
+    /**
+     * Gets the info for an existing stream, and include subject info.
+     * @param streamName the stream name to use.
+     * @param subjectFilter the subject filter to use. Can be wildcard. Null is equivalent to all subjects, the same as '>'
+     * @throws IOException covers various communication issues with the NATS
+     *         server such as timeout or interruption
+     * @throws JetStreamApiException the request had an error related to the data
+     * @return stream information
+     */
+    StreamInfo getStreamInfoWithSubjectInfo(String streamName, String subjectFilter) throws IOException, JetStreamApiException;
```

Comment:
looking for suggestions for a better name for this. I didn't want to overload getStreamInfo b/c the behavior is different, namely the original does not get any subject info